engine/src/contract_methods/connector/mod.rs

```diff
@@ -199,7 +202,10 @@ pub fn finish_deposit<I: IO + Copy, E: Env, H: PromiseHandler>(
     #[cfg(not(feature = "ext-connector"))]
     let result = internal::finish_deposit(io, env, handler)?;
     #[cfg(feature = "ext-connector")]
-    let result = external::finish_deposit(io, env, handler)?;
+    let result = {
```

Comment:
I am a little bit surprised that we redirected the `finish_deposit`, it was private method